express-delete/server.js

```diff
@@ -36,7 +36,7 @@ app.delete('/api/grades/:id', (req, res) => {
     delete grades[req.params.id];
   }
 
-  res.sendStatus(204);
+  res.sendStatus();
```

Comment:
No, sorry. This is even worse. Now you are sending whatever Express decides is the default value.

Send a 204 if you successfully delete something. Send a 404 if you can't find the id they sent you.